backend/src/controllers/profileController.js

```diff
@@ -0,0 +1,30 @@
+export default class ProfileController {
+    static profiles = []; // TODO: replace with database
+
+    static getProfiles(req, res) {
+        res.json(ProfileController.profiles);
+    }
+
+    static getProfile(req, res) {
+        const userId = req.params.id;
+        const user = ProfileController.profiles.find(user => user.id === userId);
+        if (user) {
+            res.json(user);
```

Comment:
It's not worth of returning everything that we have for this user, therefore we need to restrict returning fields

Reply:
done, unless we should be using a more elegant solution, but less readable solution